router/src/lib.rs

```diff
@@ -455,6 +455,8 @@ struct ChatCompletionRequest {
     // Additional parameters
     // TODO(travis): add other LoRAX params here
     response_format: Option<ResponseFormat>,
+    repetition_penalty: Option<f32>,
```

Comment:
Looks like the OpenAI spec defines this as `presence_penalty`: https://platform.openai.com/docs/api-reference/chat/create

Since there is no `top_k` equivalent in OpenAI, adding it as a new param is fine.